presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -118,6 +125,10 @@ public String explain(CreateTable statement, List<Expression> parameters)
                 if (columns.containsKey(name)) {
                     throw new SemanticException(DUPLICATE_COLUMN_NAME, column, "Column name '%s' specified more than once", column.getName());
                 }
+                if (!(column.isNullable() || connectorMetadata.supportsNotNullColumns())) {
```

Comment:
revert the condition:

`!column.isNullable() && !connectorMetadata.supportsNotNullColumns()`

Or even better would be:

`column.isNotNullable() && !connectorMetadata.supportsNotNullColumns()`